plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/testing/ResourceHudiTablesInitializer.java

```diff
@@ -155,6 +155,7 @@ private static void copyDir(Path srcDir, Path dstDir)
     public enum TestingTable
     {
         HUDI_NON_PART_COW(nonPartitionRegularColumns()),
+        HUDI_NON_PART_COW_CL_SC(schemaChangedNonPartitionedRegularColumns()),
```

Comment:
What's the meaning of CL and SC?

Reply:
CL means "clustered", while SC means "schema change" in this context.

Reply:
I don't think code readers can understand the meaning. Please avoid abbreviations. 
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-abbreviations

Reply:
Ok, I will remove the schema change tests altogether as requested anyways, and will change this as well in the next commit.

Reply:
I have removed the abbreviation, the table is called `HUDI_NON_PART_COW_CLUSTERING` (where `COW` is a well-known Hudi concept).